test/sampler-vec.jl

```diff
@@ -73,40 +76,42 @@ include("common.jl")
         end
         @test all_same
     end
-    @info "Adaptation tests for HMCDA with StepSizeAdaptor are skipped"
+    @info "Tests for `DenseEuclideanMetric` are skipped."
+    @info "`FixedLength` is NOT compatible with `JitteredLeapfrog`."
+    @info "Adaptation tests for `FixedLength` with `StepSizeAdaptor` are skipped."
 
     # Simple time benchmark
     let metricT=UnitEuclideanMetric
-        τ = StaticTrajectory(lf, n_steps)
+        κ = HMCKernel(Trajectory(lf, FixedNSteps(n_steps)), MetropolisTS)
 
-        time_mat = Vector{Float64}(undef, n_chains_max)
+        time_vec = Vector{Float64}(undef, n_chains_max)
         for (i, n_chains) in enumerate(n_chains_list)
             h = Hamiltonian(metricT((D, n_chains)), ℓπ, ∂ℓπ∂θ)
-            t = @elapsed samples, stats = sample(h, τ, θ_init_list[i], n_samples; verbose=false)
-            time_mat[i] = t
+            t = @elapsed samples, stats = sample(h, κ, θ_init_list[i], n_samples; verbose=false)
+            time_vec[i] = t
         end
 
         # Time for multiple runs of single chain
-        time_seperate = Vector{Float64}(undef, n_chains_max)
+        time_sep = Vector{Float64}(undef, n_chains_max)
```

Comment:
`time_seperate` is more readable than `time_sep`.

Reply:
This is a typo, should be `time_seq` (for sequential) or maybe `time_loop`.